pycbc/cosmology.py

```diff
@@ -23,14 +23,104 @@
 #
 """
 This modules provides functions for computing cosmological quantities, such as
-redshift.
+redshift. This is mostly a wrapper around ``astropy.cosmology``.
+
+Note: in all functions, ``distance`` is short hand for ``luminosity_distance``.
+Any other distance measure is explicitly named; e.g., ``comoving_distance``.
 """
 
+import astropy.cosmology
+from astropy import units
+
 import numpy
 import lal
 from scipy import interpolate
 from pycbc.conversions import ensurearray, formatreturn
 
+DEFAULT_COSMOLOGY='Planck15'
+
+def get_cosmology(cosmology=DEFAULT_COSMOLOGY):
+    """Gets an astropy cosmology class based on the given string.
+
+    Parameters
+    ----------
+    cosmology : str, optional
+        The name of the cosmology to use. Must be the name of a cosmology
```

Comment:
remove 'Must be the name ..' sentence, think it's clear without (maybe just add 'For the list of *possible* options ..'